salvador/ViewModels/GenerateImageViewModel.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  ContentView-ViewModal.swift
+//  salvador
+//
+//  Created by Geury Roustand on 13.02.23.
+//
+
+import Foundation
+import MapKit
+import SwiftUI
+
+
+
+
+//extension GenerateImageView  {
+@MainActor class GenerateImageViewModel: ObservableObject {
+    
+    @Published  var isLoading: Bool  = false
+    @Published var imageData: UIImage? = nil
+    
+    @Published var errorString: String = ""
+    @Published var showingAlert: Bool = false
```

Comment:
I wouldn't use 'ing' form for booleans. Suggestion: `showAlert` or `isAlertShown`

Reply:
U are right make more senses